test/itemmodel.test.js

```diff
@@ -31,14 +31,14 @@ describe('item tests', function () {
       assert.throws(createError, new Error('too many characters'))
     })
 
-    xit('should throw error if no title chars for new item', function () {
+    it('should throw error if no title chars for new item', function () {
       const createError = () => {
         const item = new Item('')
       }
       assert.throws(createError, new Error('no characters'))
     })
 
-    xit('should throw error if no title chars for setting', function () {
+    it('should throw error if no title chars for setting', function () {
```

Comment:
'should throw error if no chars for setting the title' => bessere Beschreibung, aktuell verwirrt die Beschreibung